### DISCUSSION TITLE
Include pantheon-tweaks in elementary OS by default

### DISCUSSION THREAD

**Discussion by joeljosedev** [elementary/triage#115] [Ideas]
It would be really useful to ship pantheon-tweaks with elementary OS because it would allow new users to easily tweak their Pantheon desktop to look the way they want it to. When I first began using elementary OS, I was not aware of pantheon-tweaks so I did spend a lot of time looking for something similar to GNOME Tweaks that I could use to customise Pantheon. So including pantheon-tweaks in elementary OS by default would save new users a lot of time and trouble.

**Comment by jeremypw** [comment]
All supported "tweaks" should already be exposed in the UI or in System Settings.  If not, then please raise individual issues for consideration.  Any unsupported tweak may rely on a setting that may be removed or changed with little warning.

**Reply by cassidyjames** [reply]
This is the correct answer. :smile: Going further, it should be clarified that Pantheon Tweaks is a third-party package that relies on changing what are often undocumented and unsupported settings—it's not something made or supported by elementary.

**Comment by danirabbit** [comment]
Gonna close as resolved as per Jeremy’s comment. Thanks!
